Dockerfile

```diff
@@ -60,12 +60,13 @@ COPY --from=build-nodejs /app/saleor/static /app/saleor/static
 COPY --from=build-nodejs /app/webpack-bundle.json /app/
 COPY --from=build-nodejs /app/templates /app/templates
 WORKDIR /app
-USER 1001
 RUN \
+    SECRET_KEY=$(cat /proc/sys/kernel/random/uuid | sed 's/-//g') \
```

Comment:
I'm guessing that if we restart the server, we lose all the user session since the secret key would have been changed?

Reply:
This image is designed to be run in a kubernetes cluster and the user is expected to set `SECRET_KEY` as part of the kubernetes secrets.

If the even that the user does not do that, then kubernetes generates it's own secret that will change between deployments - again this is the users responsibility.

As for the SECRET_KEY in the Dockerfile, it really has no purpose other than to prevent the docker build breaking with `SECRET_KEY` missing error.

It could have been left as `dummy` which it was before and make no difference in the end.

Reply:
The relevant helm chart code can be found on links below:

https://github.com/stephenmoloney/saleor-helm/blob/e461878df097cd681578953478fe51528e49cf90/deployment/helm/templates/secrets.yaml#L30-L41

![image](https://user-images.githubusercontent.com/12668653/50226840-29a56f00-039c-11e9-9c66-4e6ee3fbdfa2.png)


https://github.com/stephenmoloney/saleor-helm/blob/2226152023cee5fee50157bfea08c180a131e8a5/deployment/helm/templates/_helpers.tpl#L143-L152

![image](https://user-images.githubusercontent.com/12668653/50226817-1abebc80-039c-11e9-8b5c-5a369be34a67.png)


The README should say 

```
| `saleor-secret-key`                | Saleor/saleor                 | The django secret for the saleor application                           |  :heavy_check_mark:         |
```

I have a PR to update that now.

Reply:
Although, your comment does make me question whether this should be there at all:

```
RUN \
    SECRET_KEY=$(cat /proc/sys/kernel/random/uuid | sed 's/-//g') \
    python3 manage.py collectstatic --no-input && \
    useradd --non-unique --uid 1001 --gid 0 --create-home saleor && \
    mkdir -p /app/media /app/static && \
    chown -R 1001:0 /app
```

and instead just:

```
RUN \
    useradd --non-unique --uid 1001 --gid 0 --create-home saleor && \
    mkdir -p /app/media /app/static && \
    chown -R 1001:0 /app
```

and run 

```
python3 manage.py collectstatic --no-input
```

in the helm chart only